com.github.akiraux.libgtkcanvas.json

```diff
@@ -0,0 +1,35 @@
+{
+   "app-id": "com.github.akiraux.libgtkcanvas",
+   "runtime": "org.gnome.Platform",
```

Comment:
Any reason for not using org.freedesktop.Platform instead? If there are any, can you please update it to 3.32 so we can get this merged? Otherwise org.freedesktop.Platform/org.freedesktop.Sdk should be enough